packages/notebook/src/widget.ts

```diff
@@ -1084,6 +1086,33 @@ export namespace StaticNotebook {
     readonly raw: Record<string, any>;
   }
 
+  /**
+   * The renderer class for static notebooks.
+   */
+  class Renderer extends WindowedList.Renderer<ICellModel> {
+    createOuter() {
+      return document.createElement('div');
+    }
+
+    createViewport() {
+      const el = document.createElement('div');
+      el.setAttribute('role', 'feed');
+      el.setAttribute('aria-label', 'Cells');
+      return el;
+    }
+
+    createScrollbarItem(index: number, item: ICellModel): HTMLLIElement {
```

Comment:
Given that this receives the cell model, the list item it generates can be more interesting than just a number at some point in the future.